src/tigerbeetle/main.zig

```diff
@@ -379,6 +379,9 @@ const Command = struct {
                 .clients_limit = clients_limit,
             },
             .grid_cache_blocks_count = args.cache_grid_blocks,
+            .tracer_options = .{
+                .writer = if (args.trace) std.io.getStdOut().writer().any() else null,
```

Comment:
* aren't we closing over a pointer to a temporary here again? 
* the stdout _is_ buffered, right?